solar-land/scss/layout/_header.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../../shared/scss/components/button";
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1rem 2rem;
+  background-color: #fff;
+
+  .logo {
+    img {
+      height: 40px;
```

Comment:
As mentioned in code convention page , it is better to use rem or em for consistency

Reply:
Ok